src/utils/apiEndpoint.js

```diff
@@ -10,7 +10,7 @@ const getApiEndpoint = (location) => {
       return 'http://localhost:3000';
     }
 
-    return 'https://api.scp.biomage.net';
+    return process.env.API_ENDPOINT_PRODUCTION || 'https://api.scp.biomage.net';
```

Comment:
don't set `'https://api.scp.biomage.net'` as default -- if the endpoint is not defined, raise an error. We don't want some non-Biomage deployments to start misbehaving weirdly, because they are pointing to Biomage.